common/src/main/java/com/lightstep/tracer/shared/AbstractTracer.java

```diff
@@ -53,23 +45,24 @@
     }
 
     private final int verbosity;
-    private final Auth auth;
-    private final Runtime runtime;
+    private final Auth.Builder auth;
+    private final Reporter.Builder reporter;
+    private final CollectorClient client;
 
     /**
      * False, until the first error has been logged, after which it is true, and if verbosity says
      * not to log more than one error, no more errors will be logged.
      */
     private boolean firstErrorLogged = false;
-    private URL collectorURL;
 
     // Timestamp of the last recorded span. Used to terminate the reporting
     // loop thread if no new data has come in (which is necessary for clean
     // shutdown).
     private final AtomicLong lastNewSpanMillis;
-    private ArrayList<SpanRecord> spans;
+    private ArrayList<Span> spans;
```

Comment:
Oh, so the java classes are auto-generated from the proto file and Span is named so in that file. I don't think it's a good idea to change the name of an autogenerated file?